### PR TITLE
Adjusted file format names

### PR DESCRIPTION
**Pull request by brother**
The "explanatory names" of the file formats was not super accurate and helpful, tried to clean it up some.

### CODE REVIEW

**Comment by brother** [comment]
That @coveralls report is a bit hard to follow when you have to copy and paste the broken URL together by hand.

https://coveralls.io/builds/11961701

**Comment by tgturner** [comment]
@brother what about something like this?

```md
* Office Open XML Format Spreadsheets (xlsx, xlsm), commonly produced by newer versions of Excel
* Open Document Format Spreadsheets (ods, fods), commonly produced by LibreOffice / OpenOffice
```

or something along those lines? Do you think that would give us the best of both worlds?

**Comment by brother** [comment]
That's a variant indeed. oo.o is pretty much dead though. Not sure the application thing is that important. That's why I removed them in my proposal.

**Comment by brother** [comment]
Giving up.
